storm/server_client.go

```diff
@@ -82,9 +71,45 @@ func (c *ServerClient) Status(id string) (*ServerStatus, error) {
 	params := ServerParams{UniqID: id}
 
 	err := c.Backend.CallIntoInterface("v1/Storm/Server/status", params, &result)
-	if err != nil {
-		return nil, err
+
+	return &result, err
+}
+
+// Stop a storm server.
+func (c *ServerClient) Stop(uniqId string, force ...bool) (*ServerStop, error) {
+	var result ServerStop
+	args := map[string]interface{}{
+		"uniq_id": uniqId,
+	}
+
+	if len(force) > 0 {
+		args["force"] = force[0]
 	}
+	err := c.Backend.CallIntoInterface("bleed/server/shutdown", args, &result)
+
+	return &result, err
+}
+
+// Reboot a storm server.
+func (c *ServerClient) Reboot(uniqId string) (*ServerReboot, error) {
+	var result ServerReboot
+	args := map[string]interface{}{
+		"uniq_id": uniqId,
+	}
+
+	err := c.Backend.CallIntoInterface("bleed/storm/server/reboot", args, &result)
+
+	return &result, err
+}
+
+// Start a Storm Server.
+func (c *ServerClient) Start(uniqId string) (*ServerStart, error) {
+	var result ServerStart
+	args := map[string]interface{}{
+		"uniq_id": uniqId,
+	}
+
+	err := c.Backend.CallIntoInterface("bleed/server/start", args, &result)
 
-	return &result, nil
+	return &result, err
```

Comment:
is it not safer to return nil for the result if there's an error?  I assume you can't trust the data in `result` if there's an error, but it's possible someone might ignore the error

Reply:
changed it to that method 3d637bc34b8ddc1875c90539f334d0fddeefe698